topi/python/topi/x86/__init__.py

```diff
@@ -5,6 +5,7 @@
 from .conv2d import schedule_conv2d, schedule_conv2d_nhwc
 from .binarize_pack import schedule_binarize_pack
 from .binary_dense import schedule_binary_dense
+from .dense import *
```

Comment:
Could you also add `from batch_matmul import schedule_batch_matmul` in the __init__.py? I forgot to add it as well.